cmake/host-tools.cmake

```diff
@@ -1,5 +1,13 @@
 include($ENV{ZEPHYR_BASE}/cmake/host-tools-${ZEPHYR_GCC_VARIANT}.cmake OPTIONAL)
 
+# This allows to find 'conf' below if the Zephyr 0.9.2 SDK is installed,
+# even if we are using another toolchain (like ISSM)
+if(NOT $ENV{ZEPHYR_SDK_INSTALL_DIR} STREQUAL "")
+  list(APPEND CMAKE_PROGRAM_PATH $ENV{ZEPHYR_SDK_INSTALL_DIR}/sysroots/x86_64-pokysdk-linux/usr/bin)
```

Comment:
Is this going to work on all host operating systems?

Reply:
Nope, but windows gets the path set automagically somewhere in prebulilt_host_tools -- this is deft'ly Unixy only. Because of the path append, it does not conflict with PREBUILT_HOST_TOOLS settings

Reply:
I'd rather not rely on assumptions like this.  Also, will this work on a Mac?

Reply:
This also works and does not make any kind of assumption where things are supposed to be located: https://github.com/lpereira/zephyr/commit/e0bba46ba53c05e84d660525325528ab62226059

Reply:
Down for that, please submit PR, so we can merge that instead of this hack. Thanks @lpereira

Reply:
Done in #4854.